Pitch actions port

meganindya opened the pull request:
- `pitch`
- `scalar step`
- `nth modal pitch`
- `pitch number`
- `hertz`
- `sharp`
- `flat`
- `accidental`
- `scalar transpose`
- `semi-tone transpose`
- `register`
- `invert`
- `number to octave`
- `number to pitch`
- `set pitch number offset`
- `output tools`
- `scalar change in pitch`
- `change in pitch`
- `scalar step up`
- `scalar step down`

walterbender commented:
scalar transposition seems to have its sign flipped: transpose +1 takes G to F instead of A. -1 takes G to A instead of F.

walterbender commented:
Also, one typo on L755:          * Returns change in pithc or scalar change in pitch.

meganindya commented:
> scalar transposition seems to have its sign flipped: transpose +1 takes G to F instead of A. -1 takes G to A instead of F.

The `scalar transpose` block was hidden and so I'd skipped it. Added the corresponding action and fixed the bug.

walterbender commented:
Ah. It was hidden because it only shows up on the palette in macros. I'll retest with your changes.

walterbender commented:
Something is not correct with set pitch number offset... Investigating.

walterbender commented:
It is a typo in the args being passed into the action...
